buildscripts/cspc-operator/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# This Dockerfile builds a recent cspc-operator using the latest binary from
+# cspc-operator  releases.
+#
+FROM alpine:3.6
+
+RUN apk add --no-cache \
```

Comment:
Do we need this?

Reply:
No sure for all -- just followed the way it is for local pv.